src/pages/EmergencyViewPage/EmergencyViewRoom.tsx

```diff
@@ -1,11 +1,13 @@
-import React from "react";
+import React, { useMemo } from "react";
 import { faUserFriends } from "@fortawesome/free-solid-svg-icons";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 
 import { Room } from "types/rooms";
 
 import { useRoom } from "hooks/useRoom";
 
+import { getFirebaseStorageResizedImage } from "../../utils/image";
```

Comment:
and move it between "types" and "hooks" import groups.
```suggestion
import { getFirebaseStorageResizedImage } from "utils/image";
```

Reply:
good catch. Need to reconfigure my IDE.
https://github.com/sparkletown/sparkle/pull/2128/commits/8fc9e9becd300251bca5d4ca064d768167299581